airbyte-commons-server/src/test/java/io/airbyte/commons/server/converters/JobConverterTest.java

```diff
@@ -60,226 +46,276 @@
-
-  private static final JobDebugRead JOB_DEBUG_INFO =
-      new JobDebugRead()
-          .id(JOB_ID)
-          .configId(JOB_CONFIG_ID)
-          .status(io.airbyte.api.model.generated.JobStatus.RUNNING)
-          .configType(JobConfigType.SYNC)
-          .airbyteVersion(airbyteVersion.serialize())
-          .sourceDefinition(sourceDefinitionRead)
-          .destinationDefinition(destinationDefinitionRead);
-
-  private static final JobWithAttemptsRead JOB_WITH_ATTEMPTS_READ = new JobWithAttemptsRead()
-      .job(JOB_INFO.getJob())
-      .attempts(JOB_INFO.getAttempts().stream().map(AttemptInfoRead::getAttempt).collect(Collectors.toList()));
-
-  private static final io.airbyte.config.AttemptFailureSummary FAILURE_SUMMARY = new io.airbyte.config.AttemptFailureSummary()
-      .withFailures(Lists.newArrayList(new FailureReason()
-          .withFailureOrigin(FailureOrigin.SOURCE)
-          .withFailureType(FailureType.SYSTEM_ERROR)
-          .withExternalMessage(FAILURE_EXTERNAL_MESSAGE)
-          .withStacktrace(FAILURE_STACKTRACE)
-          .withTimestamp(FAILURE_TIMESTAMP)))
-      .withPartialSuccess(PARTIAL_SUCCESS);
-
-  @BeforeEach
-  public void setUp() {
-    jobConverter = new JobConverter(WorkerEnvironment.DOCKER, LogConfigs.EMPTY);
-    job = mock(Job.class);
-    final Attempt attempt = mock(Attempt.class);
-    when(job.getId()).thenReturn(JOB_ID);
-    when(job.getConfigType()).thenReturn(JOB_CONFIG.getConfigType());
-    when(job.getScope()).thenReturn(JOB_CONFIG_ID);
-    when(job.getConfig()).thenReturn(JOB_CONFIG);
-    when(job.getStatus()).thenReturn(JOB_STATUS);
-    when(job.getCreatedAtInSecond()).thenReturn(CREATED_AT);
-    when(job.getUpdatedAtInSecond()).thenReturn(CREATED_AT);
-    when(job.getAttempts()).thenReturn(Lists.newArrayList(attempt));
-    when(attempt.getAttemptNumber()).thenReturn(ATTEMPT_NUMBER);
-    when(attempt.getStatus()).thenReturn(ATTEMPT_STATUS);
-    when(attempt.getOutput()).thenReturn(Optional.of(JOB_OUTPUT));
-    when(attempt.getLogPath()).thenReturn(LOG_PATH);
-    when(attempt.getCreatedAtInSecond()).thenReturn(CREATED_AT);
-    when(attempt.getUpdatedAtInSecond()).thenReturn(CREATED_AT);
-    when(attempt.getEndedAtInSecond()).thenReturn(Optional.of(CREATED_AT));
-    when(attempt.getFailureSummary()).thenReturn(Optional.of(FAILURE_SUMMARY));
+  @Test
+  void testEnumConversion() {
+    assertTrue(Enums.isCompatible(JobConfig.ConfigType.class, JobConfigType.class));
+    assertTrue(Enums.isCompatible(JobStatus.class, io.airbyte.api.model.generated.JobStatus.class));
+    assertTrue(Enums.isCompatible(AttemptStatus.class, io.airbyte.api.model.generated.AttemptStatus.class));
+    assertTrue(Enums.isCompatible(FailureReason.FailureOrigin.class, io.airbyte.api.model.generated.FailureOrigin.class));
 
   }
 
-  @Test
-  void testGetJobInfoRead() {
-    assertEquals(JOB_INFO, jobConverter.getJobInfoRead(job));
-  }
+  @Nested
+  class TestJob {
```

Comment:
⚠️ The diff for this file is very messy, because I took the all but one of the existing tests and tested them into `TestJob` so that I could also create a new `TestSynchronousJob`, since the entire `setup()` was different. 

TLDR of changes: 
* testEnumConversion() is a base level test
* all other old tests into `TestJob`, all constants except for ones shared with `TestSynchronousjob` also go under `TestJob`
* new test in `TestSynchronousJob`

+ new test, no changes to old tests.